package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "lab1-devon",
+  "version": "1.0.0",
+  "description": "the first lab for CF401JS",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "test"
```

Comment:
Read the **Config** parts of the labs carefully. Remember to add scripts for lint and test. Refer to the Class 3 lecture repo into the folder called lab-duncan. Look at Duncan's package.json to see what that means.

Adding those scripts makes is to that a user who pulls down your file doesn't have to install dependencies globally on their machine. On my end, I can just type ```npm run test``` in my terminal instead of ```mocha```.